check_source.py

```diff
@@ -147,11 +150,22 @@ def check_source_submission(
             self.review_messages['declined'] = f'May not modify a non-source package of type {kind}'
             return False
 
-        if not self.allow_source_in_sle and self.sle_project_to_check:
-            if entity_exists(self.apiurl, self.sle_project_to_check, target_package):
+        if not self.allow_source_in_sle:
+            if self.sle_project_to_check and entity_exists(self.apiurl, self.sle_project_to_check, target_package):
                 self.review_messages['declined'] = ("SLE-base package, please submit to the corresponding SLE project."
                                                     "Or let us know the reason why needs to rebuild SLE-base package.")
```

Comment:
```suggestion
                self.review_messages['declined'] = ("SLFO package, please submit change to the corresponding project."
                                                    "Alternatively please provide a reason for rebuilding this SLFO package.")
```

Reply:
I think we could use SLFO as the package could be from SLES, SL Micro or similar.  Not sure if it helps the maintainer as SLFO is not exactly common term in community. Alternatively we could say just SLES as SLED is going away anyway.